app/src/main/java/org/greenstand/android/TreeTracker/view/TextButton.kt

```diff
@@ -328,6 +335,43 @@ fun DepthSurfaceCircle(
     }
 }
 
+@Composable
+fun ImageCaptureCircle(
+    modifier: Modifier,
+    color: Color,
+    shadowColor: Color,
+    ){
+    Canvas(modifier = modifier.background(shape = CircleShape, color = color)
+
+        ){
+        drawCircle(
+            color = shadowColor,
+            radius = 50f ,
+            center = Offset(
+                x = size.width  / 2,
+                y = size.height / 2
+            ),
+            style = Stroke(
+                width = 5.dp.toPx()
+            )
+        )
+        drawCircle(
+            color = shadowColor,
+            radius = 30f,
+            center = Offset(
+                x = size.width / 2,
+                y = size.height / 2
+            ),
+
+        )
+
+
+
+    }
+
+
+}
```

Comment:
Please be careful with your formatting. I've noticed a lot of extra spacing being added in your PRs.